web-ui/view/documentation.ml

```diff
@@ -0,0 +1,90 @@
+open Tyxml.Html
+
+let getting_started =
+  Template_v1.instance
+    [
+      div
+        ~a:[ a_class [ "grid grid-cols-6" ] ]
+        [
+          div
+            ~a:[ a_class [ "blog-sidebar" ] ]
+            [
+              div
+                ~a:[ Tyxml_helpers.x_data "{submenu: false}" ]
+                [ (* this is where we would have the sidebar links if we had any *) ];
+            ];
+          div
+            ~a:[ a_class [ "prose col-span-4" ] ]
+            [
+              h1 [ txt "Getting Started" ];
+              p [ txt "OCaml-CI is a CI service for OCaml projects." ];
+              hr ();
+              p
+                [
+                  txt
+                    "It uses metadata from the project's opam and dune files \
+                     to work out what to build, and uses caching to make \
+                     builds fast. It automatically tests projects against \
+                     multiple OCaml versions and OS platforms.";
```

Comment:
Can't we add here, that these versions are the ones allowed by the opam file?

Reply:
We sure can. As mentioned, this PR starts by bringing over the words that are on https://github.com/apps/ocaml-ci -- I expect that it will change over time, especially as we introduce a different sign up process.

I will add some more words as suggested.

Reply:
Sure! Thanks for the clarification :smile: